src/System.ValueTuple/src/System/ValueTuple/ValueTuple.cs

```diff
@@ -249,7 +249,7 @@ internal static int CombineHashCodes(int h1, int h2)
             // Forward to helper class in Common for this
             // We keep the actual hashing logic there, so
             // other classes can use it for hashing
-            return HashHelpers.Combine(h1, h2);
+            return HashHelpers.Combine(HashHelpers.Combine(h1, h2), HashHelpers.RandomSeed);
```

Comment:
Isn't this going to end up incorporating the random seed multiple times into hash codes?  For example, now the function:
```C#
internal static int CombineHashCodes(int h1, int h2, int h3) 
{
    return CombineHashCodes(CombineHashCodes(h1, h2), h3);
}
```
is essentially:
```
internal static int CombineHashCodes(int h1, int h2, int h3) 
{
    h4 = HashHelpers.Combine(HashHelpers.Combine(h1, h2), HashHelpers.RandomSeed);
    return HashHelpers.Combine(HashHelpers.Combine(h4, h3), HashHelpers.RandomSeed);
}
```